pkg/ebpf/netlink/conntracker.go

```diff
@@ -220,6 +224,11 @@ func (ctr *realConntracker) register(c ct.Conn) int {
 	ctr.Lock()
 	defer ctr.Unlock()
 
+	if len(ctr.state) >= ctr.maxStateSize {
+		log.Warnf("exceeded maximum conntrack state size: %d entries", ctr.maxStateSize)
+		return 0
```

Comment:
should we track this in an expvar?

Reply:
I think state size is already tracked as an expvar per @shang-wang's changes

Reply:
Oh you mean the limit itself. It likely wont be configured by customers without direct feedback from our team so I'm fine leaving it out

Reply:
I meant the number of dropped entries. It's not terribly important since we have gets/deletes/and the state size though.